molecule/pxc/pxc-innovation-lts-bootstrap-install/tasks/main.yml

```diff
@@ -0,0 +1,161 @@
+    when: ((ansible_os_family == "RedHat") and (ansible_distribution != "Amazon"))
+
+  - name: flush iptables
+    iptables:
+      flush: yes
+    when: ansible_os_family == "RedHat"
+
+  - name: install PXC {{ major_release_version}} deb packages
+    apt:
+      name: "{{ packages }}"
+      update_cache: yes
+    vars:
+      packages:
+      - percona-xtradb-cluster-full
+      - percona-toolkit
+    when: ansible_os_family == "Debian"
+    environment:
+      PERCONA_TELEMETRY_URL: "https://check-dev.percona.com/v1/telemetry/GenericReport"
+
+  - name: install PXC 8 rpm packages
+    yum:
+      name: "{{ packages }}"
+      state: latest
+    vars:
+      packages:
+      - percona-xtradb-cluster-full
+      - percona-toolkit
+    when: ansible_os_family == "RedHat"
+    environment:
+      PERCONA_TELEMETRY_URL: "https://check-dev.percona.com/v1/telemetry/GenericReport"
+
+  - name: check telemetry (enabled)
+    command: /package-testing/check_telemetry.sh pxc -e
+
+  - name: get telemetry_uuid content
+    command: cat /usr/local/percona/telemetry_uuid
+    register: t_uuid_output
+
+  - name: output telemetry_uuid content
+    debug:
+      var: t_uuid_output
+
+  - name: stop mysql service with service command
+    command: systemctl stop mysql
+
+  - name: copy pxc config on centos
+    template: src=../../../../templates/my_rpm_80.j2 dest=/etc/my.cnf
+    when: ansible_os_family == "RedHat"
+
+  - name: copy pxc certs
+    copy:
+      src: ../../../../support-files/certs/
+      dest: /etc/mysql/certs/
+
+  - name: configure PXC on debian/ubuntu
+    template: src=../../../../templates/my_8.j2
+              dest=/etc/mysql/my.cnf
+              mode=0640 owner=mysql group=root
+    when: ansible_os_family == "Debian"
+
+  # bootstrap the cluster
+  - name: bootstrap cluster on CentOS
+    command: systemctl start mysql@bootstrap.service
+
+  # set root password on centos
+  - name: set root password on centos
+    command: /package-testing/setpass_57.sh
+    when: ansible_os_family == "RedHat"
+
+  - name: copy .my.cnf with credentials on centos
+    template: src=../../../../templates/my_57.j2
+              dest=/root/.my.cnf
+              mode=0640 owner=root group=root
+    when: ansible_os_family == "RedHat"
+
+  # other tests
+# - name: install plugins
+#   command: /package-testing/plugins_test_80.sh pxc
+
+  - name: register PXC version Molecule
+    shell: mysqld --version
+    register: pxc_version
+
+  - name: print PXC version Molecule
+    debug: var=pxc_version
+
+  - name: check PXC {{ major_release_version}} version Molecule
+    command: /package-testing/version_check.sh pxc{{ major_release_version}}
+    when: lookup('env', 'check_version') == "yes" or lookup('env', 'check_version') == ""
+  
+  - name: Perform PXC package prechecks
+    shell: chmod +x /package-testing/scripts/pxc-package-prechecks.sh && /package-testing/scripts/pxc-package-prechecks.sh pxc{{ major_release_version}}
```

Comment:
It is better to put script with exec rights at once than update them during the playbook execution